lib/game.rb

```diff
@@ -0,0 +1,26 @@
+# Create a method to pull data from game.csv
+# and turn each row into a hash
+# Push that hash into a games array
+
+class Game
+  attr_reader :game_id, :season
```

Comment:
Let's try holding off on the attr_reader until we get errors that prompt us to write them.